force-app/main/default/classes/GooglePlacesUtility.cls

```diff
@@ -4,46 +4,49 @@ public with sharing class GooglePlacesUtility {
     public static void validateInputs(Id account, Decimal longitude, Decimal latitude, Integer radius, String type) {
 
         if (!type.equalsIgnoreCase('school')) {
-            AuraHandledException e = new AuraHandledException('Type needs to be school.');
-            e.setMessage('Type needs to be school.');
-            throw e;
+            throwError('Type needs to be school.');
         }
 
         if (longitude == null || latitude == null) {
-            AuraHandledException e = new AuraHandledException('Account Longitude and/or latitude are invalid.');
-            e.setMessage('Account Longitude and/or latitude are invalid.');
-            throw e;
+            throwError('Account Longitude and/or latitude are invalid.');
```

Comment:
@sungysang error label change: "Invalid Account Billing Address. Billing Address Country and Zip are missing."